frontend/src/static/js/main.js

```diff
@@ -64,6 +64,17 @@ window.deregisterMouseMove = () => {
   window.$('app-wrapper').style['user-select'] = 'auto';
 };
 
+/* global Vue hljs */
+Vue.directive('hljs', {
+  inserted(ele, binding) {
+    const element = ele;
+    const fileExtension = binding.value.split('.').pop();
+
+    element.className = fileExtension;
```

Comment:
Minor nit, just set element.className to binding.value.split dun need make new var